tools/backups2datalad/datasetter.py

```diff
@@ -77,15 +77,11 @@ def update_from_backup(
             to_save.append(d.identifier)
             self.ensure_github_remote(ds, d.identifier)
             self.tag_releases(d, ds, push=self.config.gh_org is not None)
-            if changed and self.config.gh_org is not None:
-                log.info("Pushing to sibling")
-                ds.push(to="github", jobs=self.config.jobs)
             if self.config.gh_org is not None:
-                stats = self.get_dandiset_stats(ds)
-                self.get_github_repo(f"{self.config.gh_org}/{d.identifier}").edit(
-                    description=self.describe_dandiset(d, stats)
-                )
-                ds_stats.append(stats)
+                if changed:
+                    log.info("Pushing to sibling")
+                    ds.push(to="github", jobs=self.config.jobs)
+                ds_stats.append(self.set_dandiset_gh_metadata(d, ds))
```

Comment:
set'ers which are also get'ers are "fun" especially if they `get` something else from what they `set` , but I don't have an immediate recommendation on how to RF ATM so let's let it be.